frontend/modules/common/Api.js

```diff
@@ -0,0 +1,56 @@
+import request from 'superagent';
```

Comment:
Is this API file a general place for all superagent calls? If it is, It might get a bit crowed after as we add more features. What are your thoughts on moving this into the modules action folder and have one api file for each module?